component/Card/index.html

```diff
@@ -0,0 +1,264 @@
+        <div class="card-body">
+          <h3 class="card-title">Title</h3>
+          <small class="small-text">secondary text</small>
+          <p class="card-description">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Voluptatem quis, expedita ab saepe voluptatum ut ad sequi, quasi dolores enim animi fugit nostrum autem quaerat libero doloremque! Impedit, asperiores dolores.</p>
+          <img class="badge-icon" src="/component/Card/icon/close.png" alt="badge.png">
+        </div>
+      </div>
+    </div>
+    <!-- card with badge -->
+    <div class="card">
+      <div class="card-background">
+        <img
+        class="card-img"
+        src="https://www.planetware.com/wpimages/2020/01/india-in-pictures-beautiful-places-to-photograph-gateway-of-india-mumbai.jpg"
+        alt="img"
+      />
+          <div class="card-body">
+            <h3 class="card-title">Title</h3>
+            <small class="small-text">secondary text</small>
+            <p class="card-description">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Voluptatem quis, </p>
+          </div>
+        <img class="badge-icon" src="/component/Card/icon/like.png" alt="badge.png">
+      </div>
+    </div>
+    <!-- card with shadow -->
+
+    <div class="card ">
+      <div class="card-background shadow">
+        <img
+        class="card-img"
+        src="https://www.planetware.com/wpimages/2020/01/india-in-pictures-beautiful-places-to-photograph-gateway-of-india-mumbai.jpg"
+        alt="img"
+      />
+          <div class="card-body">
+            <h3 class="card-title">Title</h3>
+            <small class="small-text">secondary text</small>
+            <p class="card-description">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Voluptatem quis, </p>
+          </div>
+        <img class="badge-icon" src="/component/Card/icon/share.png" alt="badge.png">
+      </div>
+    </div>
+
+
+
+
+    <!-- basic card -->
+    <div class="card__container">
+      <div class="card__wrapper">
+        <div class="card_background">
+          <img
+          class="image"
+          src="https://www.planetware.com/wpimages/2020/01/india-in-pictures-beautiful-places-to-photograph-gateway-of-india-mumbai.jpg"
+          alt="img"
+        />
+        <div class="card__content">
+          <h3 class="title">Our Changing Planet</h3>
+          <p class="small_content">by Kurt Wagner</p>
+          <p class="description">
+            Visit ten places on our planet that are undergoing the biggest
+            changes today.
+          </p>
+        </div>
+        </div>
+       
+        <footer class="footer">
+          <button class="btn">READ</button>
+          <button class="btn">BOOKMARK</button>
+          <div class="icons">
+            <img class="icon" src="./icon/like.png" alt="" />
+            <img class="icon" src="./icon/share.png" alt="" />
+            <img class="icon" src="./icon/three-dot.png" alt="" />
```

Comment:
You can use font-awesome icons or material Icons.

Font awesome -> 

Use the below css link in head 
```html
 <link
      rel="stylesheet"
      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css"
    />
```

and search for icons here https://fontawesome.com/v5.15/icons